bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -231,6 +231,256 @@
 					"reference": "event"
 				}
 			]
+		},
+		{
+			"uid": "core.GroupCommandTrigger",
+			"label": "a member of an item group receives a command",
+			"description": "This triggers the rule if a member of an item group receives a command.",
+			"configDescriptions": [
+				{
+					"name": "groupName",
+					"type": "TEXT",
+					"context": "item",
+					"label": "Group",
+					"description": "the name of the item group",
+					"required": true
+				},
+				{
+					"name": "command",
+					"type": "TEXT",
+					"label": "Command",
+					"description": "the received command",
+					"required": false,
+					"limitToOptions": false,
+					"options": [
```

Comment:
Might become a rather long list - I did it the same way here as we have it for all the other item triggers, so this is in line. It can be discussed whether only the most frequently used commands should be listed or all enum types, but this can be done in a separate PR if needed.

Reply:
Yes, I know. Does it make sense to add / introduce a "context": "command" for these options? Then is can be handled globally?

Reply:
Might be an option, but it would possibly shift the implementation work to the UIs then.
I would wait and see how it is really used in daily life and what we feel is missing and should be changed. I don't have a clear view on it yet.